### ISSUE TITLE
Could you please implement playing in ListView?

### ISSUE THREAD

**Issue by wann2** [chudongvip/awesome_video_player#37]
Hi, chudongvip
I think this player is very good.

I'd like to play one of videos in ListView that has lots of videos.
Like Recyclerview in Android.

**Comment by Dync-BoYuan** [comment]
> Hi, chudongvip
> I think this player is very good.
> 
> I'd like to play one of videos in ListView that has lots of videos.
> Like Recyclerview in Android.

yes, its easy to implement, you just need to create an ListView, listview contains an fake video player, when it be clicked go to the realy play page

**Comment by wann2** [comment]
Thanks for the answer.
Is it possible to play the video on the ListView in the middle of the screen, but not clicking one of them to open a new playing video page
